bond-vlan-httpks.sh

```diff
@@ -31,7 +31,7 @@ kernel_args() {
 prepare_network() {
     local tmpdir=$1
     local network=$(basename ${tmpdir})
-    echo "network:default"
+    echo "user"
     echo "network:${network}"
```

Comment:
I wonder how are we going to deal with these dynamically created networks,
(which we create to prevent static ip collisions in multiple VMs /test intances run on single hypervisor (runner)). Perhaps something for the next step ? They could be probably rather simplified using the user networking.

Reply:
Oh thanks, this bit wouldn't work. I ran quite a few networking tests in #373, but not all of them. I'll look into converting this to a user network as well. Indeed that wouldn't block this PR as long as we still run the tests on the current infra, but it will break running the tests in a container, as you can't define your own networks there.

Reply:
Done in PR #383